packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.26.0-0'
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
```

Comment:
1.26 is not added for support yet. Ideally, the chart should be tested against a new version before adding it to the kube-version list.

Reply:
@prachidamle are we fine with adding support for k8s 1.26 before its introduced in Rancher?

Reply:
yes this should be fine, as it just indicates this chart can work on k8s 1.26 cluster.